docs/packaging/msix-packaging-tool.md

```diff
@@ -72,6 +74,9 @@ on the size of the package.
     * In `<Application>` replace any existing entrypoint/trustlevel/runtimebehavior with
     `uap10:TrustLevel="appContainer" previewsecurity2:RuntimeBehavior="appSilo"`
 
+    * In `<Capabilities>` it is safe to remove the `<rescap:Capability name="runFullTrust">` unless

```

Comment:
Does comServer require runFullTrust to support running the Com Server isolated? That seems like a bug to me. Ideally the guidance should be to remove runFullTrust.

Reply:
A couple extensions do, ComServer, FirewallRules, comInterface. I've forwarded the team an email with some context from the COM team